hawkbit-dmf/hawkbit-dmf-amqp/src/test/java/org/eclipse/hawkbit/integration/AmqpServiceIntegrationTest.java

```diff
@@ -147,6 +149,14 @@ protected void assertDeleteMessage(final String target) {
         assertThat(headers.get(MessageHeaderKey.TYPE)).isEqualTo(MessageType.THING_DELETED.toString());
     }
 
+    protected void assertRequestAttributesUpdateMessage(final String target) {
```

Comment:
I wouldn't create separate methods for REQUEST_ATTRIBUTES_UPDATE topic, because it is already one-liner and the intention to check header absence/presence is clear

Reply:
this protected methods are used as wrappers for private methods.